src/app/views/query-response/pivot-items/pivot-items.tsx

```diff
@@ -130,14 +131,24 @@ export const getPivotItems = (properties: any) => {
   return pivotItems;
 };
 
-export const onPivotItemClick = (item?: PivotItem) => {
+export const onPivotItemClick = (query: IQuery, item?: PivotItem) => {
   if (!item) { return; }
   const tabTitle = item.props.title;
   if (tabTitle) {
-    telemetry.trackEvent(TAB_CLICK_EVENT, { ComponentName: `${tabTitle} tab`, QuerySignature: '' });
+    trackTabClickEvent(query, tabTitle);
   }
 };
 
+function trackTabClickEvent(query: IQuery, tabTitle: string) {
+  const normalizedUrl = normalizeQueryUrl(query.sampleUrl);
+  telemetry.trackEvent(TAB_CLICK_EVENT,
+    {
+      ComponentName: `${tabTitle} tab`,
+      QuerySignature: `${query.selectedVerb} ${normalizedUrl.incomingUrl}`,
+      QueryString: normalizedUrl.queryString
```

Comment:
I suggest we rename `QueryString` as `QueryStringSignature` and `normalized*` becomes `sanitized*`. Also specify the return type.

This comment is applicable to all functions that call trackEvent..

Reply:
Removed QueryString and concatenated it to the sanitized URL.